Gopkg.toml

```diff
@@ -34,6 +34,14 @@
   name = "github.com/kubevirt/device-plugin-manager"
 
 [[constraint]]
-  version = "~v1.10"
+  version = "v1.10"
```

Comment:
why not support all 1.10 minor version?

Reply:
@yuvalif  Had some issues with client-go syncing up. I couldn't get it to compile with all 1.10 minor version.